### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by jokedurnez**
Went through integrations 1 and 2. @kengie if you could take a look before merging?

### CODE REVIEW

File: _guides/integrations.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ OpenLattice's __Entity Data Model (EDM)__ is a blueprint meant to standardize da
 | Lives At    | Address |    
 
 
-Using either the data model details provided to you by your OpenLattice Administrator or your own interpretation of your dataset and the Entity Types that it maps to in the OpenLattice data model, found [here](https://staging.openlattice.com/edm/#/entityTypes), identify the Entity Sets you need and create them in OpenLattice's gallery dashboard. Go to the **Datasets** tab and create the new datasets with the information provided. 
+Using either the data model details provided to you by your OpenLattice Administrator or your own interpretation of your dataset and the Entity Types that it maps to in the OpenLattice data model, found [here](https://staging.openlattice.com/edm/#/entityTypes), identify the Entity Sets you need and create them in OpenLattice's gallery dashboard. Go to the **Datasets** tab and create the new datasets with the information provided.  To follow this tutorial, create a dataset of entity type `Person`.
```
Comment (kengie) [inline _guides/integrations.md]: can you modify this last sentence slightly, to, "To follow this tutorial, create a dataset for each of the Entity Types in the above table, starting with a dataset of entity type 'Person'."

File: _guides/integrations-2.md [hunk 2]
```diff
@@ -98,26 +98,6 @@ You will see your jwtToken under "account details" (not shown for privacy). Clic
...
-## 4. Read your data into java with OpenLattice's "Payload" object
-
-In `DataIntegration2018.java`, there will be some code that initializes an OpenLattice "Payload" java object to load and read your datasource from a csv.
-
-```java
-// Configure OpenLattice's Payload function to load and read your datasource
-        SimplePayload payload  = new SimplePayload( csvpath);
-```
-
 <div style="color:black; border: 1px solid black; padding: 10px; background-color: yellow; border-radius:5px; text-align: center;">Please note, if one has data from a SQL server that one would like to connect directly to, there is additional code that can read data from your server directly to OpenLattice's databases, sidestepping the need for data transfer using .csv or other files, once the proper data sharing agreements and permissions are in place. A guide for this specific situation is being written. At present, please email <a href="mailto:support@topenlattice.com">support@openlattice.com</a>  for a meeting to discuss this possibility.</div><br>
```
Comment (kengie) [inline _guides/integrations-2.md]: Here do you mind also making a change to the last two sentences in the yellow text box, change them to "The guide for this specific situation can be found in the following tutorial, 'Integrations Part 3'. Pleas email...."

File: _guides/integrations-2.md [hunk 3]
```diff
@@ -190,7 +170,7 @@ Flight csvflight = Flight.newFlight()
...
+Let's walk through how one would modify the above template for our sample dataset. Within the first `.addEntity` command, one would replace `"PEOPLE_ENTITY_SET_NAME"` with the name of the Entity Dataset that you created in step 2 of the first Integration tutorial ("Create Your Dataset"). Here, I simply call the entity alias `people` and let's assume that we previously created an Entity Dataset called `FakeJailPeople` to house the people in our data. Then, search for the "person" Entity in [**OpenLattice's data model**](https://staging.openlattice.com/edm/#/entityTypes) and replace `"LAST_NAME_FQN"` with fully qualified name of this Entity Type Property. At present, the FQN would be `nc.PersonSurName`. You can also see the FQN's in your dataset when you go to your newly created dataset in the OpenLattice platform, and you click on *View integration Details*.  Finally, replace `"LAST_NAME_COL"` with the name of the column in *your* dataset that contains last names. In our sample jail data, this column is "Last Name". The finished java code for the first entity would thus look like this:
```
Comment (kengie) [inline _guides/integrations-2.md]: Can you also remove 'staging.' from the url of the entity data model? That was an old website - it still works, but we prefer to direct people to another one.

**Comment by kengie** [comment]
Hi @jokedurnez, I made some comments and found even more old typos that we can fix now as well. Thanks!

**Comment by jokedurnez** [comment]
Made those changes !

**Comment by kengie** [comment]
great thanks!

**Pull request merged**